website/docs/developer/research/research-localstack.md

```diff
@@ -161,12 +171,16 @@ Install the AWS CLI to run CLI commands in the docker container.
     pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org awscli-local
 
 This prevents the need for specifying --endpoint-url everytime aws cli command is run. The container is now ready to test AWS services locally.
-
+:::Note
+AWSlocal is a thin wrapper around the aws command line interface for use with LocalStack.
```

Comment:
Is this wrapper installed in windows automatically or did you have to setup an alias? If you setup an alias for the wrapper, it would be good to show that here so other users know what it should be or if it is installed from somewhere, how to install it.

Reply:
I didn't setup an alias. I just installed using the pip command above and seems to work.